packages/flutter/lib/src/material/text_field.dart

```diff
@@ -762,6 +763,12 @@ class TextField extends StatefulWidget {
   ///  * [AdaptiveTextSelectionToolbar], which is built by default.
   final EditableTextContextMenuBuilder? contextMenuBuilder;
 
+  /// Determine whether this text field can request the primary focus.
+  ///
+  /// Defaults to true. Set to false if you want the [TextField] manages to do
```

Comment:
```suggestion
  /// Defaults to true. If false, the text field will not request focus when tapped, or when its context menu is displayed. If false it will not be possible to move the focus to the text field with tab key.
```
Not sure if I've got this all right. Can the developer still give the text field the focus by having its focus node request the focus? ... it looks like the answer is "no"; we should clear up that case here too.

Reply:
You are right. The text field cannot be focused if false. Sorry for missing this part.